spec/models/profile_spec.rb

```diff
@@ -323,59 +323,95 @@
     it 'activates a profile after password reset' do
       profile = create(
         :profile,
+        :password_reset,
         user: user,
-        active: false,
-        deactivation_reason: :password_reset,
       )
 
+      expect(profile.activated_at).to eq nil # to change
+      expect(profile.active).to eq false # to change
+      expect(profile.deactivation_reason).to eq 'password_reset' # to change
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at.present?).to eq false
```

Comment:
This might be clearer as either `expect(profile.gpo_verification_pending_at).to be_nil` or as `expect(profile.gpo_verification_pending_at).not_to be_present`